src/aws-sdk-helpers/cloudWatchLogs/fetchAllLogGroupsConfigurations.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  CloudWatchLogsClient,
+  LogGroup,
+  paginateDescribeLogGroups,
+} from '@aws-sdk/client-cloudwatch-logs';
+import { ARN, build, parse } from '@aws-sdk/util-arn-parser';
+import { filterCloudWatchLogsFromResources } from './filterCloudWatchLogsFromResources';
+
+const fetchAllLogGroupsConfiguration = async (
+  client: CloudWatchLogsClient,
+): Promise<LogGroup[] | undefined> => {
+  const logGroups = [];
+  for await (const page of paginateDescribeLogGroups({ client }, {})) {
+    logGroups.push(...(page.logGroups ?? []));
+  }
+
+  return logGroups.map(logGroup => ({
+    ...logGroup,
+    arn: logGroup.arn?.replace(/:\*/, ''),
```

Comment:
peut être un petit commentaire "format the arn the same way as tagging client does"